examples/graph/tool_requires/different_options/run_example.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+from test.examples_tools import run, tmp_dir
+
+
+run("conan create gcc -o myoption=1")
+run("conan create gcc -o myoption=2")
+
+output = run("conan create wine")
+assert "MYGCC=1!!" in output
+assert "MYGCC=2!!" in output
```

Comment:
This example will run agains release/2.0 branch and the latest release (2.0.4), I think we should omit it in the 2.0.4 because it will fail and remove the if after releasing

Reply:
Checking the Jenkinsfile structure, it seems a bit dirty to exclude this, maybe we should just wait until 2.0.5 is out, then update it?

Reply:
We can do that in the run_example.py, let me push the changes